activesupport/lib/active_support/concern.rb

```diff
@@ -104,6 +104,12 @@ def initialize
       end
     end
 
+    class UnreachableIncludedBlock < StandardError #:nodoc:
+      def initialize
+        super "Cannot add an 'included' block after Concern was already included"
```

Comment:
Also wouldn't this issue be resolved by explicitly requiring the needed class? Should we mention that here?